tests/unit/test_browse.py

```diff
@@ -298,39 +300,46 @@ def test_create_browse_imagery_with_mocks(
         )
 
     def test_convert_singleband_to_raster_without_colortable(self):
-        ds = MagicMock(DataArray)
-        ds.__getitem__.return_value = self.data
+        """Tests convert_gray_1band_to_raster."""
+
+        return_data = np.copy(self.data).astype('float64')
+        return_data[0][1] = np.nan
+        ds = DataArray(return_data).expand_dims('band')
 
         expected_raster = np.array(
             [
                 [
-                    [0, 104, 198, 255],
+                    [0, 0, 198, 255],
```

Comment:
Did you change this from 104 to 0 to demonstrate transparent pixel behavior?

Reply:
Well, yes, this is the expected array, I added a missing value at the [0,1] location, and these changes are the ones reflected in the output from the missing data.

Reply:
Oh right. You changed the code so the expected array changed.